build.gradle

```diff
@@ -213,6 +213,10 @@ cargo {
         } else if (activeSpringProfiles.contains("debug") || Boolean.valueOf(System.getProperty("xdebug"))) {
             jvmArgs = String.format("%s -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005", jvmArgs)
         }
+        String tlsDebug = System.getProperty("javax.net.debug");
```

Comment:
can we put it into debug / xdebug option from spring.profiles ?

Reply:
I thought about this, however, there are so many different values (it's not on/off) 

https://www.ibm.com/docs/en/sdk-java-technology/8?topic=troubleshooting-debugging-utilities

therefore, it seems simpler to be able to specify 

```
./gradlew run -Djavax.net.debug=ssl,otheroptions
```